bofire/strategies/botorch/base.py

```diff
@@ -89,10 +89,10 @@ class BotorchBasicBoStrategy(PredictiveStrategy):
     descriptor_method: DescriptorMethodEnum = DescriptorMethodEnum.EXHAUSTIVE
     categorical_encoding: CategoricalEncodingEnum = CategoricalEncodingEnum.ORDINAL
     categorical_method: CategoricalMethodEnum = CategoricalMethodEnum.EXHAUSTIVE
-    model_specs: Optional[Tmodelspecs]
-    objective: Optional[MCAcquisitionObjective]
-    acqf: Optional[AcquisitionFunction]
-    model: Optional[GPyTorchModel]
+    model_specs: Optional[Tmodelspecs] = None
```

Comment:
Why do you add the None here?

Reply:
`None` is the default value. Without `None` the static type checker does not know what the default is and it wants you to specify `model_specs=None` in the constructor of the strategy.

Reply:
Ok!